scripts/a11y/process_a11y.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import json
+
+from collections import defaultdict
+from statistics import mean
+
+
+class A11yProcessor(object):
+    ERRORS = {
+        '1_1': 'Alt Tag Errors',
+        '1_3': 'Form Errors',
+        '1_4': 'Color Contrast Errors',
+        '4_1': 'HTML Attribute Errors'
+    }
+
+    BRANCHES = {
```

Comment:
i believe that's how this works -- this is just a whitelist -- it trys to do the branch lookup (given agency) and if it fails it defaults to executive